### DISCUSSION TITLE
#_NOTES not working

### DISCUSSION THREAD

**Discussion by jebunen** [liedekef/events-made-easy#258] [Q&A] [answered]
Hi! 

I'm having problems showing the events description. Everything else seems to work when using the default single event format except the event description when using the #_NOTES placeholder. I made a template that should show only the description of the event but instead it shows this text: 
_This page is used by Events Made Easy. Don’t change it, don’t use it in your menu’s, don’t delete it. Just make sure the EME setting called ’Events page’ points to this page. EME uses this page to render any and all events, locations, bookings, maps, … anything. If you do want to delete this page, create a new one EME can use and update the EME setting ’Events page’ accordingly._

Did I accidentally broke something? Any ideas how to fix this? Many thanks in advance!

**Comment by liedekef** [comment]
How/where are you using this template? And a demo?
Also, read this on the special EME events page: https://www.e-dynamics.be/wordpress/category/documentation/3-installation-and-usage/

**Reply by jebunen** [reply, marked as answer]
I managed to fix it. I'm using Elementor and for some reason the events page was shifted to be edited with it. Weirdly it broke only the #_NOTES placeholder! Thanks again!
